packages/app-store/_utils/throwIfNotHaveAdminAccessToTeam.ts

```diff
@@ -0,0 +1,21 @@
+import getUserAdminTeams from "@calcom/features/ee/teams/lib/getUserAdminTeams";
+import { HttpError } from "@calcom/lib/http-error";
+
+export const throwIfNotHaveAdminAccessToTeam = async ({
+  teamId,
+  userId,
+}: {
+  teamId: number | null;
+  userId: number;
+}) => {
+  if (!teamId) {
+    return;
+  }
+  const teamsUserHasAdminAccessFor = await getUserAdminTeams({ userId, includeCredentials: true });
```

Comment:
I don't see why it needs includeCredentials:true here?

Reply:
Copy paste. Will fix 😄